### PR TITLE
feat: textarea

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

File: libs/docs/src/components/common/TextArea.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import { GoATextArea } from '@abgov/react-components'
+
+<Meta title="Components/TextArea" />
+
+### Component
+# TextArea
+
+[Figma design documentation](https://www.figma.com/file/3pb2IK8s2QUqWieH79KdN7/Components---DIO-DS?node-id=458%3A16042)
+
+## Basic
```
Comment (bluekurinji) [inline libs/docs/src/components/common/TextArea.stories.mdx:12]: To be consistent with other component story page, please add `<Story name="Basic">`

**Review by bluekurinji — CHANGES_REQUESTED** [review]

File: libs/web-components/src/components/text-area/TextArea.svelte [hunk 2]
```diff
@@ -8,21 +8,24 @@
...
   export let disabled: string;
 
   $: isDisabled = toBoolean(disabled);
 
   function onChange(e: Event) {
-    const target = e.target as HTMLInputElement;
+    const target = e.target as HTMLTextAreaElement;
     const value = target.value;
 
+    if (disabled) return;
```
[the remark this answers is not in the document]
Comment (chrisolsen) [reply libs/web-components/src/components/text-area/TextArea.svelte:21]: It is :) It was unexpected, but the event was firing when I was testing it, which is why it was added.

File: libs/web-components/src/components/text-area/TextArea.svelte [hunk 3]
```diff
@@ -31,12 +39,16 @@
 
 <!-- HTML -->
 <textarea
+  id={id || name}
```
Comment (bluekurinji) [inline libs/web-components/src/components/text-area/TextArea.svelte:42]: Can we make it consistent as done in checkbox. id as "id-" + name ? just to be consistent?

**Review by bluekurinji — APPROVED** [review]

**Pull request merged**